plugin/src/generators/code-checks/files/.eslint.ronasit.json.template

```diff
@@ -28,7 +28,7 @@
       }
     }
   },
-  "plugins": ["@typescript-eslint", "unused-imports", "react", "react-hooks", "react-native", "import"],
+  "plugins": ["@typescript-eslint", "@stylistic/js", "unused-imports", "react", "react-hooks", "import"],
```

Comment:
As ESLint Stylistic replaces formatting rules also from `@typescript-eslint/eslint-plugin` and `eslint-plugin-react`, let's change them in our config

Migration guide: https://eslint.style/guide/migration
Rules: https://eslint.style/packages/default#rules

Reply:
updated